lib/worker/src/db/publications.rs

```diff
@@ -137,6 +148,21 @@ pub async fn pub_cids(
     Ok(rows)
 }
 
+pub async fn find_job_cache_path_by_cid(pool: &PgPool, cid: Cid) -> Result<Option<CachePath>> {
```

Comment:
new method to return `cache_path` so `GET /records/{record_id}` can download the record from cache